src/Maestro/Maestro.Web/DarcRemoteFactory.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Maestro.Data;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.Kusto;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Maestro.Web
+{
+    public class DarcRemoteFactory : IRemoteFactory
```

Comment:
>DarcRemoteFactory [](start = 17, length = 17)

It's very odd this is called the "DarcRemoteFactory" when it is, in fact, a MaestroRemoteFactory.

Reply:
I called it DarcRemoteFactory because that is what we call it in SubscriptionActorService. Since we didn't need 90% of the stuff that SubscriptionActorService needs (git, etc), I created a new DarcRemoteFactory here. @mmitche would be able to say more about why it is called DarcRemoteFactory.

Reply:
The one in SubscriptionActorService is SUPER dangerous. I'd rather not have 2 classes, one of which is super dangerous, and one of which is 100% safe have the same name.  It will be very hard to know which class is being used without figuring out which DLL the project is referencing.  In general having 2 classes with the same name isn't a great idea, because it makes is really confusing as to which is being used.

Reply:
> The one in SubscriptionActorService is SUPER dangerous. I'd rather not have 2 classes, one of which is super dangerous, and one of which is 100% safe have the same name. It will be very hard to know which class is being used without figuring out which DLL the project is referencing. In general having 2 classes with the same name isn't a great idea, because it makes is really confusing as to which is being used.

The one in SubscriptionActor service needs access to the repos. This one does not. That's the primary difference. And it's not *super dangerous*. It's very restricted in the things it can read/write.

Reply:
Ok. The other one took out our entire service fabric and needs access to read and write files to and from the harddrive. This one does not, which is a pretty significant difference in behavior.  I don't think having a different name so that the we can tell which of the two is being used is too much to ask.

Reply:
Right exactly. That was the solution Michelle and I came up with re: isolating the pieces of remote. i think it's a pretty elegant solution actually, and fits well with the rest of things.